src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Utils.cs

```diff
@@ -681,6 +681,7 @@ internal static X509Certificate2Collection BuildBagOfCerts(KeyInfoX509Data keyIn
                         }
                     }
                     catch (CryptographicException) { }
+                    catch (PlatformNotSupportedException) { }
```

Comment:
@danmosemsft This was an untested codepath blind-ported from netfx.  Do you have a suggestion of what comment you might like?  (If it was a subtle bug, I'd agree, but this was "the feature explodes on Linux")